packages/common/src/auditlogs/AuditLogs.tsx

```diff
@@ -235,15 +261,7 @@ export const AuditLogs = () => {
             { title: 'Type', sortable: false, render: TypeDescriptor },
             { title: 'Changed', sortable: false, render: ChangeDescriptor },
             { title: 'More details', sortable: false, render: ChangeDetailsDescriptor },
-            { title: 'Time', sortable: true, render: TimeWrapper },
-            {
-              title: '',
-              sortable: false,
-              render: (item, index) => {
-                const allowsExpansion = !!item.change || item.action.includes('terminal') || item.action.includes('portforward');
-                return allowsExpansion ? <ViewDetails item={item} index={index} onIssueSelection={onIssueSelection} /> : <div />;
-              }
-            }
+            { title: 'Time', sortable: true, render: TimeWrapper }
           ]}
```

Comment:
this would break the Mender alignment - not sure if the expansion column exists in Alvaldi & CFEngine - maybe we have to make this optional then...

Reply:
@mzedel hm, I copied this from Mender https://github.com/mendersoftware/mender-server/blob/main/frontend/src/js/components/auditlogs/AuditLogs.tsx#L257-L264

Reply:
ah right, sorry I should have checked this!